datahub-web-react/src/app/preview/DefaultPreviewCard.tsx

```diff
@@ -260,9 +260,13 @@ export default function DefaultPreviewCard({
     };
 
     const shouldShowRightColumn = (topUsers && topUsers.length > 0) || (owners && owners.length > 0);
+    const uniqueOwnerUrns = new Set();
+    const uniqueOwners = owners?.filter(
+        (owner) => !uniqueOwnerUrns.has(owner.owner.urn) && uniqueOwnerUrns.add(owner.owner.urn),
+    );
 
     return (
-        <PreviewContainer data-testid={dataTestID} onMouseDown={onPreventMouseDown}>
+        <PreviewContainer data-testid={dataTestID} onMouseDown={onPreventMouseDown} id={`entityUrn-${urn}`}>
```

Comment:
Pretty sure it's a good practice to keep id's unique on the page. I this with setup, it'll render this DefaultPreviewCard along with a SearchResult at the same time?

Looking at containers search result rendering here: https://github.com/joshuaeilers/datahub/blob/779d692cdc7a74ff85f33960ff7572521736329b/datahub-web-react/src/app/entity/container/ContainerEntity.tsx#L140

and then again down below in the SearchResultList.

I could be wrong here about the multiple times, but it might be a good idea to put this into a class or other data-* attribute so we don't collide.

Reply:
ah yeah good call they would definitely both be rendered with the same ID. will make that change to a class or something else. I really only think we need one of them anyways...